### DISCUSSION TITLE
How to jump to file from search results?

### DISCUSSION THREAD

**Discussion by recmo** [zed-industries/zed#17489] [Feature Requests]
When searching, finding references, or looking at issues, Zed turns into this 'snippets-buffer-mode'. How do I go from a line into this mode to viewing the file at that location? The little 'Jump to file' arrow just takes me to the file, but not to that line. I also can not find the keyboard shortcut for it. It's also not in the context menu.

Right now I need to memorize the line number, and manually navigate there.

**Comment by bennetbo** [comment]
Hey, the 'snippets-buffer-mode' is called a multi buffer. 
Here's the relevant [docs](https://zed.dev/docs/multibuffers#navigating-to-the-source-file) section, which explains how to open an excerpt at a specific line

**Comment by schpet** [comment]
`alt-enter` is the keyboard shortcut for `editor: open excerpts`

**Reply by theonly1me** [reply]
That's `option-enter` for MacOS users.

**Reply by alvinleung** [reply]
Is there a setting for closing buffer panel after opening the except? It would be nice so that there won't be millions of Definition buffers opened.

<img width="751" alt="Screenshot 2025-03-12 at 10 32 38 AM" src="https://github.com/user-attachments/assets/6d757ea3-adbe-427a-b3ab-0918b9ece2ac" />
